docs/sources/enterprise/enterprise-configuration.md

```diff
@@ -361,6 +361,19 @@ This value limits the size of a single cache value. If a cache value (or query r
 
 The default is `1`.
 
+## [caching.encryption]
+### enabled
+
+Setting 'enabled' to `true` allows users to encrypt the query values in the cache.
+
+This value is `false` by default.
```

Comment:
remove

Reply:
I updated it to be "The default is `false`.", but kept it on a different line to follow the pattern of the other settings in this markdown. Does that seem okay?